CMSgov/easi-app
# EASI-1608 - Some i18n Keys are rendering raw templates

# EASI-1608

## Changes and Description

- Wrapped text in trans component to prevent i18n Keys from rendering raw templates
- Reviewed i18n translation files to make sure text with link templates use same component

## How to test this change

1. Create a 508 testing request with no documents in it
2. Click the 508 testing request to load the main page
3. Review text under "Next step: Provide your documents" - link text should be "templates page (opens in a new tab)" without showing <1></1> raw template

## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
